WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -100,6 +102,30 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 				wallet.Synchronizer.UsdExchangeRate, false);
 		}
 
+		ChangeAvoidanceSuggestionViewModel? bnbSuggestion = null;
+
+		BranchAndBound bnb = new(wallet.Coins.Confirmed().ToList());
```

Comment:
`wallet.Coins.Confirmed().ToList()`

I'm really not sure, if this is the correct way to get all of your available coins or not.

Reply:
What else would it be, any ideas?